app/controllers/volunteers_controller.rb

```diff
@@ -1,9 +1,11 @@
 class VolunteersController < ApplicationController
   include NestedAttributes
+  include VolunteerAttributes
   before_action :set_volunteer, only: [:show, :edit, :update, :destroy]
 
   def index
-    @volunteers = Volunteer.all
+    @q = Volunteer.ransack(params[:q] || { state_cont: 'registered' })
```

Comment:
Well, fine. No problem. But I save this as a credit for bailing the process next time I like to ;-)